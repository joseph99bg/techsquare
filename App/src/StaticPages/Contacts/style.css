.contacts-page {
    text-align: center;
    margin: 30px auto 0 auto;
}
.contacts-page h1 {
    font-family: 'Orbitron';
    color: #20aaea;
    font-weight: 600;
    font-size: 40px;
    margin-bottom: 50px;
}
.contacts-page h3 {
    font-family: 'Orbitron';
    color: #717ED5;
    font-size: 25px;
}
.contacts-page i {
    color: #717ED5;
    font-size: 25px;;
}
.contacts-page span {
    font-size: 20px;
    font-family: 'Montserrat';
    font-weight: 400;
    color: #000;
    margin-bottom: 20px;
    display: block;
}
.contacts-page span a {
    color: #717ED5;
    text-decoration: none;
}
.contacts-page span a:hover {
    text-decoration: underline;
}
.contacts-page iframe {
    margin-top: 50px;;
}