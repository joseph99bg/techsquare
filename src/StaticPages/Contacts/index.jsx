import React from 'react'
import './style.css'

function Contacts() {
    return (
        <div className="contacts-page">
            <h1>Contacts Page</h1>
            <div className="half-col">
                <h3>Address and contacts:</h3>
                <div className="adress-info">
                    <span>
                        <i className="fas fa-map-pin"></i> Sliven, Bulgaria, Hristo Botev 27-A-12<br/>
                    </span>
                    <span>
                        <i className="fas fa-phone"></i> <a href="tel:0882745300">0882 74 53 00</a>
                    </span>
                </div>
            </div>
            <div className="half-col">
            <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d1466.469204165047!2d26.32464638535452!3d42.68384698844658!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x40a6274608247037%3A0xd4ffb16c34fa1837!2z0LHRg9C7LiDigJ7QpdGA0LjRgdGC0L4g0JHQvtGC0LXQsuKAnCAyN9CQLCA4ODAxINCh0LvQuNCy0LXQvSDQptC10L3RgtGK0YAsINCh0LvQuNCy0LXQvQ!5e0!3m2!1sbg!2sbg!4v1576610847274!5m2!1sbg!2sbg" width="100%" height="450" frameborder="0" allowfullscreen=""></iframe>
            </div>
        </div>
    )
}

export default Contacts