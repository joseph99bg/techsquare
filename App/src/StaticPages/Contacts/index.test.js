import React from 'react'
import Contacts from './index'
import renderer from 'react-test-renderer'

describe('Contacts component', () => {
    test('should render correctly', () => {
        const component = renderer.create(
            <Contacts />
        )
        const tree = component.toJSON()
        expect(tree).toMatchSnapshot()
    })
})