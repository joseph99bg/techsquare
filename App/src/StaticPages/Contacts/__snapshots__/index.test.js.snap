// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contacts component should render correctly 1`] = `
<div
  className="contacts-page"
>
  <h1>
    Contacts Page
  </h1>
  <div
    className="half-col"
  >
    <h3>
      Address and contacts:
    </h3>
    <div
      className="adress-info"
    >
      <span>
        <i
          className="fas fa-map-pin"
        />
         Sliven, Bulgaria, Hristo Botev 27-A-12
        <br />
      </span>
      <span>
        <i
          className="fas fa-phone"
        />
         
        <a
          href="tel:0882745300"
        >
          0882 74 53 00
        </a>
      </span>
    </div>
  </div>
  <div
    className="half-col"
  >
    <iframe
      allowfullscreen=""
      frameborder="0"
      height="450"
      src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d1466.469204165047!2d26.32464638535452!3d42.68384698844658!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x40a6274608247037%3A0xd4ffb16c34fa1837!2z0LHRg9C7LiDigJ7QpdGA0LjRgdGC0L4g0JHQvtGC0LXQsuKAnCAyN9CQLCA4ODAxINCh0LvQuNCy0LXQvSDQptC10L3RgtGK0YAsINCh0LvQuNCy0LXQvQ!5e0!3m2!1sbg!2sbg!4v1576610847274!5m2!1sbg!2sbg"
      title="contacts-map"
      width="100%"
    />
  </div>
</div>
`;
